test/SlowTests/Server/Documents/SqlMigrationTests.cs

```diff
@@ -0,0 +1,416 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Serialization;
+using FastTests;
+using Raven.Client.Documents.Operations;
+using Sparrow.Platform;
+using Xunit;
+
+
+namespace SlowTests.Server.Documents
+{
+    public class SqlMigrationTests : RavenTestBase
+    {
+        private const string DatabaseName = "SqlMigrationTestDatabase";
+
+        private static readonly string CreateDatabaseQuery = "USE master IF EXISTS(select * from sys.databases where name= '{0}') DROP DATABASE[{0}] CREATE DATABASE[{0}]";
+    
+        private static readonly string DropDatabaseQuery = "IF EXISTS(select * from sys.databases where name= '{0}') ALTER DATABASE [{0}] SET SINGLE_USER WITH ROLLBACK IMMEDIATE; IF EXISTS(select * from sys.databases where name= '{0}') DROP DATABASE [{0}]";
+
+        private static readonly Lazy<string> MasterDatabaseConnection = new Lazy<string>(() =>
```

Comment:
Pretty sure that we already have this in the Sql ETL, is there a reason to duplicate it?